exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+let greetings = "hello "
+
+function makeNameUpperCase (name){
```

Comment:
For this exercise we should write two functions that one of them nested in second one like this : 

function upperCase(name){
    
    return name.toUpperCase();
}
function showName(greet,name){
    return greet +" "+ **upperCase(name)**; // as you see here I use the first function in second function 
}

and then after that you should call the last function : showName(greet, name);